test/tpc-bench/Makefile

```diff
@@ -238,4 +238,25 @@ file-tpch-load:
 	@for table in region nation customer supplier part partsupp orders lineitem; do \
 		echo "  - name: $$table" >> ./file-tpch-spicepod/spicepod.yaml; \
 		echo "    path: file:./$$table.parquet" >> ./file-tpch-spicepod/spicepod.yaml; \
-	done
\ No newline at end of file
+	done
+
+mssql-init:
+	sqlcmd -U $(DB_USER) -P $(DB_PASS) -S $(DB_HOST) -d master -Q "IF NOT EXISTS (SELECT name FROM master.dbo.sysdatabases WHERE name = N'$(DB_NAME)') CREATE DATABASE $(DB_NAME);"
+	sqlcmd -U $(DB_USER) -P $(DB_PASS) -S $(DB_HOST) -d $(DB_NAME) -i ./mssql-tpch/up.sql
+	@echo "Database $(DB_NAME) has been successfuly created or updated."
+
+mssql-load:
+	@for table in region nation customer supplier part partsupp orders lineitem; do \
+		cp tpch-kit/dbgen/$$table.tbl ./mssql-tpch/data/$$table.tbl; \
+	done
+	chmod 755 ./mssql-tpch/data
+	sqlcmd -U $(DB_USER) -P $(DB_PASS) -S $(DB_HOST) -d $(DB_NAME) -i ./mssql-tpch/insert.sql
```

Comment:
The `dss.ddl` in TPCH kit can be reused here instead of creating a separate SQL file for initiating table

Reply:
I elected not to use it, because it doesn't create primary or foreign keys. `dss.ri` would be troublesome to reuse without some re-writing, and both files need to have `GO` added to the end of them. I settled on making a new file instead.